src/style/internal/shared_input-select-picker.scss

```diff
@@ -243,12 +240,12 @@ $height-of-helper-text-pseudo-before: 0.75rem; // There is strange a pseudo befo
                         color: var(--lime-error-text-color);
                     }
                 }
-                .trailing-icon {
+                .mdc-text-field__icon--trailing {
                     color: var(--lime-error-text-color);
                 }
             }
 
-            .mdc-text-field__icon.trailing-icon {
+            .mdc-text-field__icon--trailing.trailing-icon {
```

Comment:
https://user-images.githubusercontent.com/35954987/125762427-e617b685-a5ec-4226-933c-1b96bca6990f.mov

There are a few visual problems here in the video. The biggest one which is the absence of full-width links in readonly mode is caused by `.mdc-text-field__icon--trailing.trailing-icon`, which should have been `.mdc-text-field__icon--trailing` only. I'll push one or a couple of fixups.

Reply:
![image](https://user-images.githubusercontent.com/154725/125777857-dba29123-9c9d-47a8-8159-4769cbec67ba.png)

Wait… you made a video of a video? 🤨 😂

Reply:
hehe, yes 😸 because first I used GIPHY CAPTURE to screen record, but they seem to use a codec that makes HUUUUGE video files. So I couldn't upload that file. I recorded again using QuickTime Player. :P

Reply:
I just use the built-in tools available via `cmd-shift-5`. I think those work great 🙂

Reply:
Resolving, as this discussion is continued here instead: https://github.com/Lundalogik/lime-elements/pull/1297#discussion_r670372397